### ISSUE TITLE
facing delays in getting activities through Gnippy 

### ISSUE THREAD

**Issue by bmeshruti08** [abh1nav/gnippy#18]
Hi,

I am facing a delay of approximately 10 minutes from the time I tweet to the time I get ithe tweet in callback. Is there a way out where this delay can be avoided.

Thanks,
Shruti

**Comment by abh1nav** [comment]
@bmeshruti08 Hi Shruti - I haven't experienced this yet, but it's possible that streams could experience delays on GNIP's end from time to time. There's no special code for buffering in `gnippy` so I'm not sure what I can do for you here. Can you provide a bit more information about how you're using it along with some sample code? Thanks!

**Comment by bmeshruti08** [comment]
Hi,

I just wrote this test script. Initially I was using something similar and there were not delays but recently I started to face delays and with this script upon tweeting I see the activity in  script after 10 minutes. So a total of 10 minutes of delay is there.
I have attached the test script below. I also reported to Gnip and with the curl command to retireve the activities 
![test_monitoring](https://cloud.githubusercontent.com/assets/20816402/17427865/a44d2cc0-5b01-11e6-8e94-4f7afebfecae.jpg)

they provided to test for delay there was actually a delay of 1 second which means no delay for me.

Thanks,
Shruti
